### PR TITLE
feat: add option to set default array merge operation

### PR DESCRIPTION
**Pull request by Makatumba**
This is the feature implementation requested in #29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -257,6 +260,15 @@ Object that will be available in [`$expression`](#expression) operations as `$pa
 
 Set this property to `true` to stringify the JSON result. Set the property to `"pretty"` if the output should be pretty printed.
 
+### `defaultArrayMergeOperation: "combine" | "replace" | "concat"`
+
+Set this property to override default merge operation.
+Default value is set to [`"concat"`](#concat). Possible values are:
```
Comment (boschni) [inline README.md:266]: Think the default is `combine`?

File: bin/json-merger.js [hunk 2]
```diff
@@ -15,6 +15,10 @@ program
     "-op, --operation-prefix [prefix]",
     "the operation prefix. Defaults to $"
   )
+  .option(
+    "-am, --array-merge [operation]",
```
Comment (boschni) [inline bin/json-merger.js:19]: Maybe good to use `--default-array-merge-operation` for consistency

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "json-merger",
-  "version": "1.1.4",
+  "version": "1.1.5",
```
Comment (boschni) [inline package.json:3]: I'll update this one in a release commit

File: __tests__/operations/combine.test.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+const jsonMerger = require("../../dist");
+const { testConfig } = require("../../__helpers__");
+
+describe("when merging two objects and a source property has a $combine indicator", function () {
```
Comment (boschni) [inline __tests__/operations/combine.test.js:4]: Could you also add some tests to check what happens if you try to use combine on other values like objects?

**Comment by boschni** [comment]
Great work! Added some small comments

**Comment by Makatumba** [comment]
thank you for your feedback =D. I've applied requested changes

**Comment by boschni** [comment]
Awesome! Thanks for your contribution

**Pull request merged**